2024-03-13 20:16:36,458 - regression_class_conspiracy_logistic - INFO - Running FSS
2024-03-13 20:16:38,604 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,

2024-03-13 20:16:38,604 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)

2024-03-13 20:16:38,608 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,

2024-03-13 20:16:38,608 - py.warnings - WARNING - /home/cara/anaconda3/envs/redditenv/lib/python3.10/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)

2024-03-13 20:16:38,613 - regression_class_conspiracy_logistic - INFO - Model 1
2024-03-13 20:16:38,681 - regression_class_conspiracy_logistic - INFO - Model 2
2024-03-13 20:16:38,866 - regression_class_conspiracy_logistic - INFO - Model 3
2024-03-13 20:16:39,068 - regression_class_conspiracy_logistic - INFO - Model 4
2024-03-13 20:16:39,368 - regression_class_conspiracy_logistic - INFO - Model 5
2024-03-13 20:16:39,620 - regression_class_conspiracy_logistic - INFO - Model 6
2024-03-13 20:16:39,824 - regression_class_conspiracy_logistic - INFO - Model 7
2024-03-13 20:16:40,051 - regression_class_conspiracy_logistic - INFO - Model 8
